app/src/androidTest/java/org/wikipedia/TestLogRule.kt

```diff
@@ -0,0 +1,35 @@
+package org.wikipedia
+
+import android.util.Log
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+class TestLogRule : TestRule {
+    override fun apply(base: Statement?, description: Description?): Statement {
```

Comment:
It looks like the `base` and `description` parameters don't need to be nullable here.

Reply:
removed.